src/nni_manager/training_service/pai/paiTrainingService.ts

```diff
@@ -130,6 +130,20 @@ class PAITrainingService implements TrainingService {
         return Promise.resolve(paiTrialJob);
     }
 
+    public async getTrialLog(trialJobId: string, logType: LogType): Promise<string> {
+        const dir: string = HDFSClientUtility.getHdfsTrialWorkDir(this.paiClusterConfig!.userName, trialJobId);
```

Comment:
The log files does not exist when trial job is running for PAI,  should we check job status here?